pkg/providers/instancetype/types.go

```diff
@@ -63,15 +63,14 @@ type Resolver interface {
 	CacheKey(nodeClass *v1.EC2NodeClass) string
 	// Resolve generates an InstanceType based on raw InstanceTypeInfo and NodeClass setting data
 	Resolve(ctx context.Context, info ec2types.InstanceTypeInfo, zoneData []ZoneData, nodeClass *v1.EC2NodeClass) *cloudprovider.InstanceType
-	// UnavailableOfferingsChanged tells whether the unavailable offerings cache has been updated since the last call
-	UnavailableOfferingsChanged() bool
+	// GetUnavailableOfferingsSeqNum returns the current seq num of the unavailable offerings cache
+	GetUnavailableOfferingsSeqNum() uint64
```

Comment:
The instancetype provider could alternatively directly access the unavailable offerings cache for this, allowing this interface to remain unmodified but requiring a new field to be added to the instancetype provider constructor.